client/control/constants.js

```diff
@@ -0,0 +1,10 @@
+export const BASE_API_URL = '/api/v1/';
+
+export const API_REQUEST_SETTINGS = {
+  credentials: 'include',
+  headers: {
+    'X-CSRFToken': document.getElementsByTagName('html')[0].dataset.csrf,
```

Comment:
This looks a little fragile - probably no way to define this value as a constant somewhere, right?